sale_disable_inventory_check/models/product_category.py

```diff
@@ -0,0 +1,15 @@
+# Copyright Komit <http://komit-consulting.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductCategory(models.Model):
+    _inherit = 'product.category'
+
+    property_is_check_stock = fields.Boolean(
```

Comment:
The semantics of this field is not immediately clear to me 'is_check_stock' is not immediately. In any case, I would advise against introducing a double negation: Make it so that a negative value corresponds to Odoo not doing something.

You could rename to 'check_stock_on_sale' to make the name a little clearer (and of course change the default property value).